### ISSUE TITLE
False name clash warning on Refactor|Rename

### ISSUE THREAD

**Issue by daFreeMan** [rubberduck-vba/Rubberduck#4026]
Win10, Office2016 (desktop), RD .3280

In a class module:

```vba
Private Type TFoo
  theBar as Long
End Type

Private this As TFoo

Public Property Get Bar() as Long
  Bar = this.theBar
End Property
Public Property Let Bar(ByVal RHS as Long)
  this.theBar = RHS
End Property
```

Attempting to rename `Bar()` to `thisBar()` generates a warning. Ignoring the warning and continuing does produce a correctly renamed Getter/Letter and compilable code

![rd false name clash](https://user-images.githubusercontent.com/11889733/40365357-602fafc2-5da2-11e8-8de6-8169cd3661b2.PNG)

_Just realized the highlighting in the image is rubbish, but you get the idea..._

**Comment by MDoerner** [comment]
You want to rename to `theBar` not `thisBar`, right? The screenshot would suggest that.

**Comment by daFreeMan** [comment]
Yes, renaming `Get Bar()` to `Get theBar()`.

I see my typo now.

**Comment by comintern** [comment]
This is fixed, probably by (as of now un-merged) changes in #4274.
